modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/settingsContext.js

```diff
@@ -133,18 +133,22 @@ export function updateFieldName(
 	editingLanguageId,
 	fieldNameGenerator,
 	focusedField,
-	value
+	value,
+	invalid = false
```

Comment:
This parameter indicates that the incoming new `value` for `fieldName` (the function is named `updateFieldName`) is invalid. Do you think it would be better to name it `invalidFieldName`, `invalidValue`?

Reply:
We can use `invalidValue` 😄

Reply:
I prefer `isInvalidValue`, because this is a boolean an this is kind of a convention in Portal.

Reply:
Did it [here](https://github.com/liferay-forms/liferay-portal/pull/2488/commits/e613f38d249c0eaa73ba67296994b3835a8e1d9f).